### PR TITLE
international trainees and teachers new draft

### PR DESCRIPTION
**Pull request by EmmaFrith**

### CODE REVIEW

File: content/international-candidates.md [hunk 1]
```diff
@@ -52,217 +52,162 @@ keywords:
   - UK Ancestry Visa
 ---
 
-## Teaching or training to teach in England
+Find out how to train to become a teacher in England.
 
-Your route to working or training as a teacher in England and the financial support you might get depends on:
+[Already a teacher? Come to England to teach](/international-candidates#already-a-teacher-teach-in-england).
```
Comment (peteryates) [inline content/international-candidates.md]: Content wise this is fine but on the page it feels a bit odd being squished between the hero and the following `h2`:

Screenshot from 2021-04-09 11-52-35

Could this be another CTA in the right column perhaps?

File: content/international-candidates.md [hunk 2]
```diff
@@ -52,217 +52,162 @@ keywords:
...
-Salaried training types include:
+You do not necessarily have to be in England to study with an English institution. You can do an international Postgraduate Certificate of Education (PGCEi) outside the UK.
 
-* school direct (salaried)
-* Teach First
-* postgraduate teaching apprenticeships
+A PGCEi will help you develop your teaching skills and knowledge.
 
-An ‘unsalaried’ teacher training course means you pay a fee.
+However, it does not lead to QTS and will not qualify you to teach in schools in England which require QTS.
```
Comment (peteryates) [inline content/international-candidates.md]: I'm not sure about this but without a comma there's a hint of ambiguity as to whether all schools in England require QTS or some do and you'd only be excluded from them.

```suggestion
However, it does not lead to QTS and will not qualify you to teach in schools in England, which require QTS.
```

File: content/international-candidates.md [hunk 3]
```diff
@@ -52,217 +52,162 @@ keywords:
...
+You may have a pre-existing right to work in the UK, such as settled or pre-settled status under the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain in the UK](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk).
 
-First, [choose your teaching training provider](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+[If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
 
-Then, contact your chosen teacher training provider to ask about the school where you’ll be employed during your training. The school must be a Home Office licensed employer sponsor – sponsors can help you apply for your visa.
+### Consider getting qualified teacher status
 
-### To work as a teacher
+You need 'qualified teacher status' (QTS) to teach in many state schools in England.
```
Comment (peteryates) [inline content/international-candidates.md]: Isn't it all state schools?

**Review by peteryates — COMMENTED** [review]
Looks great and definitely a step forwards. Left a few observations and questions.

File: content/international-candidates.md (hunk 2 above)
Comment (EmmaFrith) [inline content/international-candidates.md]:
```suggestion
However, it does not lead to QTS. This means you will not be able to teach in schools which require QTS. 
```

File: content/international-candidates.md [hunk 4]
```diff
@@ -140,7 +140,7 @@ You do not necessarily have to be in England to study with an English institutio
 
 A PGCEi will help you develop your teaching skills and knowledge.
 
-However, it does not lead to QTS and will not qualify you to teach in schools in England which require QTS.
+However, it does not lead to QTS. This means you will not be able to teach in schools which require QTS.
```
Comment (EmmaFrith) [inline content/international-candidates.md:143]: change to: This means you will not be qualified to teach in state schools.

File: content/international-candidates.md (hunk 3 above)
Comment (EmmaFrith) [inline content/international-candidates.md]:
```suggestion
You need 'qualified teacher status' (QTS) to teach in state schools in England.
```

Comment (EmmaFrith) [reply content/international-candidates.md]: i think you're right - will double check with policy

File: content/international-candidates.md [hunk 5]
```diff
@@ -52,217 +58,144 @@ keywords:
...
+#### To pass certain checks
 
-### Irish citizens
+Your training provider will check you’re suitable for teaching in terms of your health, whether you're safe to work with children and your level of maths and English.
 
-Irish citizens can continue to enter, work and study in the UK under the [Commmon Travel Area](https://www.gov.uk/government/publications/common-travel-area-guidance/common-travel-area-guidance) as they did previously.
+### Get support
 
-## Choosing the correct visa for your job or course
+You can get an adviser for one-to-one support.
```
Comment (peteryates) [inline content/international-candidates.md:109]: Is this all applicants or only those for key subjects?

File: content/international-candidates.md [hunk 6]
```diff
@@ -52,217 +58,144 @@ keywords:
...
-[Check minimum rates for teachers by region in England](/salaries-and-benefits).
+<p class="call-to-action__action">
+  <a href="https://teaching-vacancies.service.gov.uk/?utm_source=int_teacher_recruitment&utm_medium=referral&utm_campaign=AY21-22">Search for a teaching <span>job</span></a>
+</p>
 
-## Ways to train or work as a teacher in the UK without a skilled worker visa or a student visa
+### Get support if you're qualified to teach maths, physics or modern languages
 
-### Graduate visa
+If you’re an international qualified teacher of maths, physics, or modern foreign languages at secondary school age, you can get one-to-one support with:
```
Comment (peteryates) [inline content/international-candidates.md]: This doesn't read right to me, 'internationally-qualified' perhaps?

**Review by peteryates — COMMENTED** [review]
This looks way better to me.

Definitely needs a third pair of eyes to check that what's come out is covered eleswhere or isn't necessary.

File: content/international-candidates.md (hunk 5 above)
Comment (MylesJarvis) [reply content/international-candidates.md:109]: For international candidates who aren't returners? All applicants. If they were a returner, the key subjects triage would apply.

File: content/international-candidates.md [hunk 7]
```diff
@@ -52,217 +58,144 @@ keywords:
...
 
-## Choosing the correct visa for your job or course
+You can get an adviser for one-to-one support.
 
-For teaching jobs in England, the main visa route is the ‘[skilled worker visa](https://www.gov.uk/skilled-worker-visa)’.
+Your adviser can help you get a 'statement of comparability' showing [how your overseas qualifications compare to UK qualifications](https://enic.org.uk/Qualifications/SOC/Default.aspx).
 
-For teacher training in England, the main routes are either the ‘[student visa](https://www.gov.uk/student-visa)’ or the ‘[skilled worker visa](https://www.gov.uk/skilled-worker-visa)’.
+<p class="call-to-action__action">
+  <a href="https://beta-adviser-getintoteaching.education.gov.uk/">Get an <span>adviser</span></a>
```
Comment (MylesJarvis) [inline content/international-candidates.md]: Need to update to /tta-service as the URL

File: content/international-candidates.md [hunk 8]
```diff
@@ -52,217 +58,144 @@ keywords:
...
-Unsalaried courses include:
+## Already a teacher? Teach in England
 
-* higher education institution programmes
-* school-centred initial teacher training (SCITT)
-* school direct (fee-funded)
+Teach in England and you’ll benefit from working in a world-class education system.
 
-[Learn more about ways to train in England](/guidance/become-a-teacher-in-england).
+You’ll earn a [competitive salary](https://getintoteaching.education.gov.uk/salaries-and-benefits), get supported in your professional development and have varied job opportunities.
```
Comment (MylesJarvis) [inline content/international-candidates.md]: /salaries-and-benefits

File: content/international-candidates.md [hunk 9]
```diff
@@ -52,217 +58,144 @@ keywords:
...
-For an unsalaried teacher training course, you will need a [student visa](https://www.gov.uk/student-visa). Student visas must be sponsored by a Home Office licensed student sponsor.
+### What you need to start teaching
 
-First, [choose your teaching training provider](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+To teach in England you need:
 
-Then, contact your chosen teacher training provider to check they are a Home Office licensed student sponsor – sponsors can help you apply for your visa.
+* teaching skills and experience
+* to pass criminal and professional safeguarding checks (these will be organised by your employer)
+* to [get the right visa or status](/international-candidates#get-the-right-visa-or-status)
```
Comment (MylesJarvis) [inline content/international-candidates.md]: I've seen links that just use the # and then the id, whereas this one has the URL as well. Which is best practice/correct @peteryates

File: content/international-candidates.md (hunk 6 above)
Comment (MylesJarvis) [reply content/international-candidates.md]: I agree w/ Pete

File: content/international-candidates.md [hunk 10]
```diff
@@ -52,217 +58,144 @@ keywords:
...
+If you’re an international qualified teacher of maths, physics, or modern foreign languages at secondary school age, you can get one-to-one support with:
 
-If you are already an international student in the UK, you can apply for a [graduate visa](https://www.gov.uk/guidance/new-immigration-system-what-you-need-to-know#international-students-and-graduates) on successful completion of your degree. This visa will be available from summer 2021 and you will need to have been sponsored by a Home Office licensed student sponsor to apply.
+* the application process, such as writing a personal statement and preparing for an interview
+* accessing courses to enhance your subject knowledge
+* finding teaching vacancies
 
-The graduate visa will allow you to study, work, or look for work in the UK for up to 2 years after completing your studies (3 years for PhD students), without a sponsor. This includes training as a teacher on a salaried or unsalaried initial teacher training course.
+<p class="call-to-action__action">
+  <a href="https://beta-adviser-getintoteaching.education.gov.uk/">Get an <span>adviser</span></a>
```
Comment (MylesJarvis) [inline content/international-candidates.md]: Needs updating

File: content/international-candidates.md [hunk 11]
```diff
@@ -52,217 +58,144 @@ keywords:
...
+</p>
 
-You will be able to switch from a graduate visa to another visa (such as the [skilled worker visa](https://www.gov.uk/skilled-worker-visa) without having to leave the UK, if you meet the requirements for the new visa.
+### Find out what salary you can earn
 
-### The Youth Mobility Scheme visa
+Schools have some flexibility on [salary](https://getintoteaching.education.gov.uk/salaries-and-benefits). It varies by region and your level of skill, experience and qualifications.
```
Comment (MylesJarvis) [inline content/international-candidates.md]: /salaries-and-benefits

File: content/international-candidates.md [hunk 12]
```diff
@@ -52,217 +58,144 @@ keywords:
...
+There are [specific salary requirements for a skilled worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).
 
-You must be 18 to 30 years old to apply. If you are successful, you’ll be able to study or work in the UK without a sponsor. This includes working as a teacher or training as a teacher via either a salaried or unsalaried initial teacher training course.
+### Learn about life as a teacher in England
 
-A Youth Mobility Scheme visa lasts up to 2 years. You may be able to switch from a Youth Mobility Scheme visa to another visa route (for example the skilled worker visa   ) without having to leave the UK. You’ll need to meet the requirements for the other visa route.
+[Read other teachers’ stories](https://getintoteaching.education.gov.uk/my-story-into-teaching) and research the [English curriculum](https://www.gov.uk/national-curriculum) and [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards).
```
Comment (MylesJarvis) [inline content/international-candidates.md]: /my-story-into-teaching

**Review by MylesJarvis — CHANGES_REQUESTED** [review]

File: content/international-candidates.md (hunk 9 above)
Comment (peteryates) [reply content/international-candidates.md]: When on the same page just the `#id`

File: content/international-candidates.md [hunk 13]
```diff
@@ -52,217 +58,144 @@ keywords:
...
 
-## Choosing the correct visa for your job or course
+You can get an adviser for one-to-one support.
 
-For teaching jobs in England, the main visa route is the ‘[skilled worker visa](https://www.gov.uk/skilled-worker-visa)’.
+Your adviser can help you get a 'statement of comparability' showing [how your overseas qualifications compare to UK qualifications](https://enic.org.uk/Qualifications/SOC/Default.aspx).
 
-For teacher training in England, the main routes are either the ‘[student visa](https://www.gov.uk/student-visa)’ or the ‘[skilled worker visa](https://www.gov.uk/skilled-worker-visa)’.
+<p class="call-to-action__action">
+  <a href="https://adviser-getintoteaching.education.gov.uk/">Get an <span>adviser</span></a>
```
Comment (peteryates) [inline content/international-candidates.md]:
```suggestion
  <a href="/tta-service">Get an <span>adviser</span></a>
```

File: content/international-candidates.md [hunk 14]
```diff
@@ -52,217 +58,144 @@ keywords:
...
+If you’re an internationally-qualified teacher of maths, physics, or modern foreign languages at secondary school age, you can get one-to-one support with:
 
-If you are already an international student in the UK, you can apply for a [graduate visa](https://www.gov.uk/guidance/new-immigration-system-what-you-need-to-know#international-students-and-graduates) on successful completion of your degree. This visa will be available from summer 2021 and you will need to have been sponsored by a Home Office licensed student sponsor to apply.
+* the application process, such as writing a personal statement and preparing for an interview
+* accessing courses to enhance your subject knowledge
+* finding teaching vacancies
 
-The graduate visa will allow you to study, work, or look for work in the UK for up to 2 years after completing your studies (3 years for PhD students), without a sponsor. This includes training as a teacher on a salaried or unsalaried initial teacher training course.
+<p class="call-to-action__action">
+  <a href="https://adviser-getintoteaching.education.gov.uk/">Get an <span>adviser</span></a>
```
Comment (peteryates) [inline content/international-candidates.md]:
```suggestion
  <a href="/tta-service">Get an <span>adviser</span></a>
```

**Review by peteryates — APPROVED** [review]
Looks great.

**Pull request merged**